vocabularies/Common.xml

```diff
@@ -553,34 +553,27 @@ The minimum structure is
-      </Term>
-      <ComplexType Name="WeakReferentialConstraintType">
-        <Annotation Term="Common.Experimental" String="for references to code lists and master data" />
-        <Property Name="ReferenceRoot" Type="Edm.String" Nullable="false">
-          <Annotation Term="Core.Description" String="URI of the service containing the reference set" />
-          <Annotation Term="Core.LongDescription" String="Need not be a uniform resource locator, for cases where the actual location of the reference service is not static or not known at design-time." />
-        </Property>
-        <Property Name="ReferencePath" Type="Edm.String" Nullable="false">
-          <Annotation Term="Core.Description" String="Resource path of an OData collection with reference values, relative to ReferenceRoot" />
-        </Property>
-        <Property Name="ForeignKeys" Type="Collection(Common.ForeignKeyType)" Nullable="false">
-          <Annotation Term="Core.Description" String="Instructions on how to construct the value list request and consume response properties" />
-        </Property>
-      </ComplexType>
-      <ComplexType Name="ForeignKeyType">
-        <Annotation Term="Common.Experimental" String="for references to code lists and master data" />
-        <Property Name="Property" Type="Edm.PropertyPath" Nullable="false">
-          <Annotation Term="Core.Description" String="Path to property of the annotated entity set" />
-        </Property>
-        <Property Name="ReferencedProperty" Type="Edm.String" Nullable="false">
-          <Annotation Term="Core.Description" String="Path to property in reference set, format is identical to PropertyPath annotations" />
+      <Term Name="SAPObjectType" Type="Common.SAPObjectTypeType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The SAP Object Type represented by the annotated entity type" />
+        <Annotation Term="Core.LongDescription">
+          <String>SAP Object Types are a generalization of Business Object, Technical Object, Configuration Object, and Analytical Object.</String>
+        </Annotation>
+      </Term>
+
+      <ComplexType Name="SAPObjectTypeType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Information about an SAP Object Type" />
+        <Property Name="Name" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="The name of the SAP Object Type" />
         </Property>
       </ComplexType>
 
+      <Term Name="SAPObjectTypeReference" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The SAP Object Type referenced by the annotated property" />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The name of the SAP Object Type referenced by the annotated property" />
```
since `SAPObjectType` is a complex type, not just a name.